addon/components/uni-autocomplete.js

```diff
@@ -1,22 +1,25 @@
 import { computed } from '@ember/object';
-import { isPresent, isBlank } from '@ember/utils';
+import { isPresent, isBlank, isEmpty } from '@ember/utils';
 import Component from '@ember/component';
 import layout from '../templates/components/uni-autocomplete';
 import ClickOutside from '../mixins/click-outside';
 import KeyCodes from 'ember-cli-uniq/enums/key-codes-type';
+import { A } from '@ember/array';
+import { capitalize } from '@ember/string';
 
 export default Component.extend(ClickOutside, {
   classNames: ['uni-autocomplete'],
   layout,
   noResultsComponent: 'uni-autocomplete-no-results',
   options: [],
   value: '',
-  placeholder: '',
   showOptions: false,
   hasError: false,
   highlighted: 0,
   maxOptionsToShow: 4,
   autocomplete: 'off',
+  noResultsText: 'No results for ',
+  placeholderText: '',
```

Comment:
Please document this breaking change, or add a backwards-compatible property

Reply:
done